ios/Classes/BetterPlayer.m

```diff
@@ -409,7 +408,9 @@ - (void)observeValueForKeyPath:(NSString*)path
         }
     } else if (context == playbackLikelyToKeepUpContext) {
         if ([[_player currentItem] isPlaybackLikelyToKeepUp]) {
-            [self updatePlayingState];
+            if (_pipController.pictureInPictureActive == false) {
+                [self updatePlayingState];
+            }
```

Comment:
This is reason PIP player freeze, only calls `updatePlayingState` when pip is not active